debian/control

```diff
@@ -0,0 +1,29 @@
+Source: myst
+Section: devel
+Priority: optional
+Maintainer: Mysterium Team <core-services@mysterium.network>
+Build-Depends:
+ dh-golang,
+ debhelper (>= 8.0.0),
+ golang-1.11,
+Standards-Version: 3.9.7
+Homepage: https://mysterium.network
+Vcs-Git: git://github.com/mysteriumnetwork/node.git
+Vcs-Browser: https://github.com/mysteriumnetwork/node
+XS-Go-Import-Path: github.com/mysteriumnetwork/node
+
+Package: myst
+Architecture: any
+Built-Using: ${misc:Built-Using}
+Depends:
+ ${misc:Depends},
+ ${shlibs:Depends},
+ sudo,
+ resolvconf,
```

Comment:
Looks like duplicated with `bin/package_debian`. 
Aren't we reusing existing .deb? Or are we going to drop .deb file with public PPA releases?

Reply:
`bin/package_debian` builds a binary DEB package, but for PPA we have to provide a source DEB package, which will be built on the Launchpad.
And yes, I think it will be possible to drop building DEB packaged and leave it only in PPA.